### ISSUE TITLE
Landing Page: Text is not programatically accessible 

### ISSUE THREAD

**Issue by swickr** [w3c/validator-suite#181]
The text "description" presented to a mouse user when they hover their mouse over any of the Heading level 3 banners in "The Marks of Success" section, is not accessible to a screen reader user since the headings are not focusable and the descriptions are hidden with <code>display: none</code>. This technique hides the content from screen readers as well. Consider hiding them by displaying them offscreen and moving them into position on hover. This will allow a screen reader user to access the text. You should also consider making the headings focusable (<code>tabindex=0</code>) and toggling the display of the "description" on focus. You would most likely have to script this behavior. Currently, a keyboard only user would not be able to expose the "description." 

See [Mark's full assessment](https://lists.w3.org/Archives/Team/team-vs/2013Aug/att-0010/ValidatorSuiteA11y.html)
